measureimagefocus_requirements.txt

```diff
@@ -0,0 +1,11 @@
+six
+scipy
+scikit-image
+numpy
+nose
+matplotlib
+click
+Pillow
+dask
+tensorflow
+microscopeimagequality
```

Comment:
`microscopeimagequality` and `matplotlib`* are the only non-CellProfiler requirements that need to be listed in this requirements file. Dependencies of these libraries should automatically be installed and don't need to be explicitly specified.

*matplotlib is a dependency of CellProfiler, and could also be omitted (assuming CellProfiler never drops matplotlib as a dependency)